totalopenstation/formats/landxml.py

```diff
@@ -0,0 +1,360 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+# filename: landxml.py
+# Copyright 2015 Damien Gaignon <damien.gaignon@gmail.com>
+#
+# This file is part of Total Open Station.
+#
+# Total Open Station is free software: you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Total Open Station is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty
+# of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Total Open Station.  If not, see
+# <http://www.gnu.org/licenses/>.
+
+import xml.etree.ElementTree as xml
+import time
+import re
+import os
+
+from . import Parser
+
+# Template file relativ to this file path
+TEMPLATE = "../../data/template.xml"
+
+DEFAULT_NS = "http://www.landxml.org/schema/LandXML-1.1"
+DECLARATION = '<?xml version="1.0" encoding="UTF-8"?>'
+
+
+def _indent(elem, level=0):
```

Comment:
Is this really needed?

I also don't like at all that it's taken from StackOverflow https://stackoverflow.com/a/4590052 without attribution or even a link.

Reply:
It is just cosmetic to easily read the XML and not having a one line serialized XML.
I will remove it if you think it is not needed.

Reply:
I made a credit for the snippet.